frontend/src/component/personalDashboard/createChartOptions.ts

```diff
@@ -0,0 +1,161 @@
+import type { Theme } from '@mui/material/styles/createTheme';
+import type { ChartOptions } from 'chart.js';
+import { formatTickValue } from '../common/Chart/formatTickValue';
+import type { ILocationSettings } from '../../hooks/useLocationSettings';
+import type { IPoint } from '../feature/FeatureView/FeatureMetrics/FeatureMetricsChart/createChartData';
+import {
+    formatDateHM,
+    formatDateYMD,
+    formatDateYMDHM,
+} from '../../utils/formatDate';
+
+const formatVariantEntry = (
+    variant: [string, number],
+    totalExposure: number,
+) => {
+    if (totalExposure === 0) return '';
+    const [key, value] = variant;
+    const percentage = Math.floor((Number(value) / totalExposure) * 100);
+    return `${value} (${percentage}%) - ${key}`;
+};
+
+export const createPlaceholderBarChartOptions = (
+    theme: Theme,
+): ChartOptions<'bar'> => ({
+    plugins: {
+        legend: {
+            position: 'bottom',
+            labels: {
+                color: theme.palette.text.primary,
+                pointStyle: 'circle',
+                usePointStyle: true,
+                boxHeight: 6,
+                padding: 15,
+                boxPadding: 5,
+            },
+        },
+        tooltip: {
+            enabled: false,
+        },
+    },
+    responsive: true,
+    scales: {
+        x: {
+            stacked: true,
+            ticks: {
+                color: theme.palette.text.secondary,
+            },
+            grid: {
+                display: false,
+            },
+        },
+        y: {
+            stacked: true,
+            ticks: {
+                color: theme.palette.text.secondary,
+                maxTicksLimit: 5,
+                callback: formatTickValue,
+            },
+            grid: {
+                drawBorder: false,
+            },
+        },
+    },
+    elements: {
+        bar: {
+            borderRadius: 5,
+        },
+    },
+    interaction: {
+        mode: 'index',
+        intersect: false,
+    },
+});
+
+export const createBarChartOptions = (
+    theme: Theme,
+    hoursBack: number,
+    locationSettings: ILocationSettings,
+): ChartOptions<'bar'> => {
+    const { plugins, responsive, elements, interaction, scales } =
+        createPlaceholderBarChartOptions(theme);
+    return {
+        plugins: {
+            legend: plugins?.legend,
+            tooltip: {
```

Comment:
tooltip only shows up when we have metrics data, it's not present in the placeholder. The styles are copied from the NetworkTrafficUsage. We may eventually create a shared tooltip styling after we get another use case (avoid premature abstraction)